src/containers/SignupContainer/form.tsx

```diff
@@ -32,8 +36,17 @@ const SignUpForm = () => {
 
   const onSubmit = (values: FormValues) => {
     console.log(isRememberChecked);
+    FetchAuth({ email: values.email, storeName: values.storeName, password: values.password }).then(
```

Comment:
rename to fetchAuth (which is a bad name btw, i mentioned above why)

Reply:
Move the logic to another file (in the same directory

Reply:
Calling the fetchAuth like this `({ email: values.email, storeName: values.storeName, password: values.password })` is messy.

Reply:
Use try/catch here